src/App.jsx

```diff
@@ -1,98 +1,13 @@
 /* eslint-disable max-len */
 
 import './App.scss';
-// import moviesFromServer from './api/movies.json';
+import { MovieList } from './components/MovieList/MovieList';
```

Comment:
```suggestion
import { MovieList } from './components/MovieList';
```

You can avoid name duplicates in the path using the helping index.js file in the directory of each component.